README.md

```diff
@@ -125,7 +127,7 @@ import {Svg} from 'react-native-svg';
 
 The following code will produce something like this:
 
-![Example code result](https://github.com/investingwolf/react-native-svg-path-gradient/blob/main/images/heart.gif)
+![Example code result](https://i.imgur.com/uM8UtLn.gifv)
```

Comment:
You pressed V without the control an extra time when pasting the link LOL

Reply:
no its supposed to be `.gifv`. A gifv is essentially an mp4 or webm file with special instructions that tell html to play it in a loop. The result is a much smaller, more capable, and faster to load file.

> Imgur started converting any GIF files uploaded to the site to video and giving the link to the HTML player the appearance of an actual file with a .gifv extension.

- https://en.wikipedia.org/wiki/GIF#Animation_formats

Reply:
oh TIL. I didn't just say that after looking at the code though - github doesn't display the image in the readme when on that branch:
![Screen Shot 2020-12-22 at 1 12 12 PM](https://user-images.githubusercontent.com/5751797/102919768-70c90d80-4457-11eb-89d2-3bcb6a2a2fa9.png)
So I looked at the link and saw the v, then tried it on imgur without the v because I thought that was weird and saw that the gif did show up.

Reply:
id switch it tp just gif. since its not showing up for me either